1.0/debian/sdk/msbuild/hooks/post_push

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+echo "Pushing additional sdk-msbuild tags"
```

Comment:
@naamunds, I am guessing you are going to call out the usage of `tags` in the comment when only one additional tag is pushed.  I am proposing plural as the number of tags we have for images are constantly changing.  Changing this comment constantly between singular and plural adds such little value IMO.  I think the comment as is conveys the intent of the code unless you have a wording that would be 100% correct and doesn't need updating between singular and plural images.